integration-tests/contract-multisig-set-delegate.spec.ts

```diff
@@ -0,0 +1,139 @@
+import { CONFIGS } from "./config";
+import { MANAGER_LAMBDA } from "@taquito/taquito";
+import { genericMultisig } from "./data/multisig";
+
+CONFIGS().forEach(({ lib, rpc, setup, createAddress }) => {
+  const Tezos = lib;
+ 
+  describe(`Generic Multisig set delegate: ${rpc}`, () => {
+    beforeEach(async (done) => {
+      await setup(true)
+      done()
+    })
+    test('test manager transfers set delegate scenarios', async () => {
+
+      const account1 = await createAddress();
+      const account2 = await createAddress();
+      const account3 = await createAddress();
+
+      const pkh = await Tezos.signer.publicKeyHash();
+      const op = await Tezos.contract.registerDelegate({});
```

Comment:
Wondering if lines 20 to 23 are necessary for the current test

Reply:
i have to say the pkh is a delegate beofre i can get the contract to add it as delegate through lambda

Reply:
oh yes, make sense!